code/modules/nifsoft/software/01_vision.dm

```diff
@@ -100,7 +100,6 @@
 	list_pos = NIF_MESONS
 	cost = 1000
 	a_drain = 0.1
-	access = access_engine
```

Comment:
You didn't add miners, you just removed access.

Reply:
I know.... I've since then changed my mind about the scope of this pull but without my old repo I can't fix it myself. I've edited the comment to original to reflect what I wanted for this now, sorry for the confusion.